sdk/data_cosmos/src/operations/list_attachments.rs

```diff
@@ -46,6 +46,7 @@ impl ListAttachmentsBuilder {
             let this = self.clone();
             let ctx = self.context.clone();
             async move {
+                let _ = &this;
```

Comment:
`cargo fix --edition` added this for the 2021 edition. Why the change? What is this doing? Can it be simplified?

Reply:
May be it is not needed.